ibis/backends/flink/__init__.py

```diff
@@ -442,3 +444,67 @@ def _get_operations(cls):
     @classmethod
     def has_operation(cls, operation: type[ops.Value]) -> bool:
         return operation in cls._get_operations()
+
+    def insert(
+        self,
+        table_name: str,
+        obj: pd.DataFrame | ir.Table | list | dict,
```

Comment:
`create_table` also accepts `pyarrow.Table` -- do we also want to accept that here?

Reply:
That's a good point! We definitely can - I think we can just convert pyarrow tables to pandas tables and insert them the same way.

FWIW though, the `insert()` method in alchemy and pyspark backends don't accept pyarrow tables.

Reply:
Added `pyarrow.Table`!